cli/tests/data/missing_params_dcos.toml

```diff
@@ -2,5 +2,5 @@
 reporting = false
 email = "test@mail.com"
 [package]
-cache = "true"
 sources = [ "git://github.com/mesosphere/universe.git", "https://github.com/mesosphere/universe/archive/master.zip",]
+cache = "true"
```

Comment:
Sorry. Side effect of running the tests (reading and saving the same file) in my local env.